rustfmt-core/rustfmt-config/src/lib.rs

```diff
@@ -88,6 +88,7 @@ create_config! {
         "Align enum variants discrims, if their diffs fit within threshold";
     match_arm_blocks: bool, true, false, "Wrap the body of arms in blocks when it does not fit on \
         the same line with the pattern of arms";
+    strip_leading_match_arm_pipes: bool, true, true, "";
```

Comment:
Good spot. I included a corresponding tip for the discussed 3-variant enum config option that we'll use instead of the boolean `strip_leading_match_arm_pipes` option